src/hotspot/share/gc/shenandoah/mode/shenandoahPassiveMode.cpp

```diff
@@ -55,9 +56,9 @@ void ShenandoahPassiveMode::initialize_flags() const {
   // Final configuration checks
   // No barriers are required to run.
 }
-ShenandoahHeuristics* ShenandoahPassiveMode::initialize_heuristics(ShenandoahGeneration* generation) const {
+ShenandoahHeuristics* ShenandoahPassiveMode::initialize_heuristics(ShenandoahHeapStats* heap_info) const {
```

Comment:
Can take out this argument now.

Reply:
Can't take it out. It's required by the interface to support initialization of adaptive heuristic.